src/GraphQL/ParseGraphQLSchema.js

```diff
@@ -64,20 +67,24 @@ class ParseGraphQLSchema {
     this.log =
       params.log || requiredParameter('You must provide a log instance!');
     this.graphQLCustomTypeDefs = params.graphQLCustomTypeDefs;
+    this.appId = params.appId;
```

Comment:
Is it worth adding the `requiredParameter` check for `appId`?

Reply:
done